app/src/main/java/nodomain/freeyourgadget/gadgetbridge/activities/charts/StatsChartFragment.java

```diff
@@ -0,0 +1,217 @@
+/*  Copyright (C) 2015-2017 0nse, Andreas Shimokawa, Carsten Pfeiffer,
```

Comment:
I only copy/paste your SleepChartFragment and did some adjustments to reach my goal :)